src/monster.h

```diff
@@ -548,6 +549,7 @@ class monster : public Creature
         cata::optional<time_point> baby_timer;
         bool biosignatures;
         cata::optional<time_point> biosig_timer;
+        time_point udder_timer;
```

Comment:
The previous approach (adding an `effect`) was sound. Can it still be used?

Reply:
Effects do not progress outside of the reality bubble.

Reply:
Isn't it fast-forwarded when the player re-enters the reality bubble. If it's not, I'd consider that a bug.